test/integration/helpers/Env.cs

```diff
@@ -23,19 +23,28 @@ private static void Init()
                     var path = AppDomain.CurrentDomain.BaseDirectory;
                     var sr = new StreamReader(path + "env.json");
                     var jsonString = sr.ReadToEnd();
-                    _env = JsonConvert.DeserializeObject<Dictionary<string, string>>(jsonString);
+                    _env = JsonSerializer.Deserialize<Dictionary<string, JsonElement>>(jsonString, new JsonSerializerOptions
+                    {
+                        PropertyNameCaseInsensitive = true
+                    });
                 }
             }
-            catch
+            catch (JsonException jsonEx)
             {
-                _env = new Dictionary<string, string>();
+                Console.WriteLine($"Error parsing JSON: {jsonEx.Message}");
```

Comment:
is it ok to mute errors here?

Reply:
We were muting them before as well, so yeah. I just added this 2 cases with proper message

Reply:
in such case you may assign the default value to _env before the `try` block to avoid unnecessary code duplication

Reply:
generally, exceptions mute is considered a suboptimal coding practice. Check, for example, https://xp123.com/code-smells-in-exceptions/

Reply:
added default values to _env before the try.
I agree that silencing expectations is not ideal, but in this case, which is just internal testing for contributors, there is no need to fail with an exception.